### ISSUE TITLE
`htaccess` format is invalid for WP CLI import via `wp redirection import .htaccess --format=htaccess`

### ISSUE THREAD

**Issue by mkopala** [johngodley/redirection#2013]
I'm trying to import redirects using the `redirection` WP CLI command.

Here's the version of the `redirection` plugin:

```
$ wp plugin list | grep redirection
redirection	active	none	4.4.2
```

Running like this gives the error:
```
$ wp redirection import .htaccess --format=htaccess
Error: Invalid import format - csv, json, or htaccess supported
```

Usage message says `htaccess` is a valid option:

![Selection_999(148)](https://user-images.githubusercontent.com/561932/66618598-7efe7700-eb8e-11e9-8dad-b6edd3991946.png)

Code in `redirection-cli.php`:

```
                $importer = Red_FileIO::create( $format );

                if ( ! $importer ) {
                        WP_CLI::error( 'Invalid import format - csv, json, or htaccess supported' );
                        return;
                }
```
![Selection_999(142)](https://user-images.githubusercontent.com/561932/66618280-6772be80-eb8d-11e9-814d-508e16a9235e.png)

But no `htaccess` option in `models/file-io.php`:

![Selection_999(143)](https://user-images.githubusercontent.com/561932/66618305-7c4f5200-eb8d-11e9-92d3-2c4dd3f8cc44.png)

Command works fine with `apache` instead:

```
$ wp redirection import .htaccess --format=apache
Success: Imported 5 redirects as apache
```

**Comment by mkopala** [comment]
I didn't quite follow the bug report template.  I hope you don't mind. :wink:

**Comment by johngodley** [comment]
Fixed in dae54530ffaa46fa7bc960ed5059c7a16a0f0928 and released in the next version. Thanks for the report.
